src/election_anomaly/munge_routines/__init__.py

```diff
@@ -10,45 +11,68 @@ class MungeError(Exception):
     pass
 
 
-def clean_raw_df(raw,munger):
-    """Replaces nulls, strips whitespace, changes any blank entries in non-numeric columns to 'none or unknown'.
-    Appends munger suffix to raw column names to avoid conflicts"""
+def generic_clean(df:pd.DataFrame) -> pd.DataFrame:
```

Comment:
Is this type of annotation/typing needed here and elsewhere? I think it's better practice to keep a codebase consistent unless there's a requirement that can't be met without it. I'm not at all opposed to moving toward a more strongly typed approach, but I think that should either be done with: 1) a concerted feature branch which tackles this throughout the codebase; or 2) slowly adding it in with new code or as we fix bugs in existing areas. While I haven't heard of a desire to move towards this approach, that doesn't mean that the desire doesn't exist. So we might want to have a larger conversation about this.

Reply:
Typing made code writing faster (via autocomplete) and debugging easier (via PyCharm's syntax checking). I'm guessing that consistent type checking would help newer folks understand the code. That said, I don't know that it's enough of a priority to warrant your suggestion 1; but your suggestion 2 is lighter weight. I could go either way.